File: SimpleHabit/app/src/main/java/com/me/simplehabit/adapters/SessionAdapter.java
package com.me.simplehabit.adapters;

import android.support.annotation.NonNull;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.me.simplehabit.R;
import com.me.simplehabit.data.vos.SessionVO;
import com.me.simplehabit.views.holders.SessionViewHolder;

public class SessionAdapter extends BaseRecyclerAdapter<SessionViewHolder, SessionVO> {

    @NonNull
    @Override
    public SessionViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
        View itemview= LayoutInflater.from(viewGroup.getContext()).inflate(R.layout.view_item_session,viewGroup,false);
        return new SessionViewHolder(itemview);
    }
}

File: SimpleHabit/app/src/main/java/com/me/simplehabit/adapters/SessionAdapter.kt
package com.me.simplehabit.adapters

import android.view.LayoutInflater
import android.view.ViewGroup

import com.me.simplehabit.R
import com.me.simplehabit.data.vos.SessionVO
import com.me.simplehabit.views.holders.SessionViewHolder

class SessionAdapter : BaseRecyclerAdapter<SessionViewHolder, SessionVO>() {

    override fun onCreateViewHolder(viewGroup: ViewGroup, i: Int): SessionViewHolder {
        val itemview = LayoutInflater.from(viewGroup.context).inflate(R.layout.view_item_session, viewGroup, false)
        return SessionViewHolder(itemview)
    }
}
